index.ts

```diff
@@ -192,48 +199,56 @@ async function run(): Promise<void> {
         }
     }
 
-    const args = []
+    if (program.fullstackExample == "saas") {
```

Comment:
Ah, I get it:
* --example is used to create a Shuttle app based on a Next.js example and bootstraps the Shuttle backend on the fly.
* --shuttle-example is used to setup the next.js app with an existing backend based on a Shuttle example.
* --fullstack-example has both, but it will provide existing templates to start from, without being able to configure them.

I like this approach. However, can we change the `if {} else {}` logic here to something of this sort:
```suggestion
fullstackExample = false
if (program.fullstackExample == "saas") {
     await cloneExample({
            repository: SHUTTLE_SAAS_URL,
            projectPath: resolvedProjectPath,
      })
      createShuttleToml(shuttleProjectName, resolvedProjectPath)
      fullstackExample = true
} else {
      // Maybe we can display also the supported types.
      console.error("The provided fullstack example is not known. Please provide a supported example.");
      return;
}

if (!fullstackExample) {
    const args = []
     ....
}

const shuttleOrange = chalk.hex("#ff8a3f")
    console.log(
        shuttleOrange(`
....
```

Reply:
Basically, yeah. Ok that's fine.

I think the main thing is that in the long term if we keep adding more and more fullstack templates this approach may be a little inflexible but at the moment it works and without knowing exactly what templates we'll add in the future or if we *will* add more, this works perfectly ok